code/game/jobs/job/marines.dm

```diff
@@ -46,7 +46,7 @@ What you lack alone, you gain standing shoulder to shoulder with the men and wom
 /datum/job/marine/engineer
 	title = "Squad Engineer"
 	comm_title = "Eng"
-	paygrade = "E4"
+	paygrade = "E3"
```

Comment:
yeah this is what you should be using for `get_paygrades`, add two vars for job datums with the short/long titles and just reference those.